privateca/snippets/src/test/java/privateca/SnippetsIT.java

```diff
@@ -73,6 +75,12 @@ public class SnippetsIT {
 
   private ByteArrayOutputStream stdOut;
 
+  int maxAttemptCount = 3;
+  int initialBackoffMillis = 300000; // 5 minutes
```

Comment:
Should we use uppercase constant names as per the style guide: https://engdoc.corp.google.com/eng/doc/devguide/java/style/index.md?cl=head#s5.2.4-constant-names

Reply:
Yep. Added!